yajsapi/agreement/config.ts

```diff
@@ -21,16 +21,18 @@ export class AgreementConfig {
   readonly api: RequestorApi;
   readonly logger?: Logger;
   readonly eventTarget?: EventTarget;
+  public readonly httpAgent: Agent;
```

Comment:
Looks like `private` to me...

Reply:
I don't think so. We use this object later in entities, eg. Agreement, but maybe the name is misleading...

`this.options.httpAgent.destroy?.();`